cpu/stm32f1/include/cpu_conf.h

```diff
@@ -65,6 +65,13 @@ extern "C" {
 #elif defined(CPU_MODEL_STM32F103RE) || defined(CPU_MODEL_STM32F103ZE)
 #define FLASHPAGE_NUMOF     (256U)
 #endif
+
+/* The minimum block size which can be written is 2B. However, the erase
+ * block is always FLASHPAGE_SIZE.
+ */
+#define FLASHPAGE_RAW_BLOCKSIZE    (2U)
+/* Writing should be always 4 bytes aligned */
+#define FLASHPAGE_RAW_ALIGNMENT    (4U)
```

Comment:
@cladmi, It could be one of those 2 lines, can you test with a value of 2 ?

Reply:
With `ENABLE_DEBUG (1)` in `stm32_common/periph/flashpage.c` it works so it must be something else.